src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1650,15 +1650,15 @@ def create_vm(vm_name, resource_group_name, image=None, size='Standard_DS1_v2',
         ]
         nic_resource = build_nic_resource(
             nic_name, location, tags, vm_name, subnet_id, private_ip_address, nsg_id,
-            public_ip_address_id)
+            public_ip_address_id, )
```

Comment:
should provide the `application_security_groups`?

Reply:
Yes, I'm still actively working on this piece, but I wanted to get it out there for early review. :)